readthedocs/doc_builder/python_environments.py

```diff
@@ -43,15 +43,20 @@ def delete_existing_build_dir(self):
     def install_package(self):
         setup_path = os.path.join(self.checkout_path, 'setup.py')
         if os.path.isfile(setup_path) and self.config.install_project:
-            if self.config.pip_install or getattr(settings, 'USE_PIP_INSTALL', False):
+            if self.config.pip_install or getattr(
+                    settings, 'USE_PIP_INSTALL', False):
+                extra_req_param = ''
+                if self.config.extra_requirements:
+                    extra_req_param = '[{0}]'.format(
```

Comment:
`{0}` is used everywhere else in the code base. For Python 2.6 compatibility I think.

Even if I can find a couple of `{}` occurrences for string formatting, the majority is `{0}`-like.

Should I switch back to `{}` anyway?